core-services/importExport/src/main/java/org/silverpeas/core/importexport/publication/PublicationContentType.java

```diff
@@ -23,16 +23,29 @@
  */
 package org.silverpeas.core.importexport.publication;
 
+import org.silverpeas.core.importexport.form.XMLModelContentType;
 import org.silverpeas.core.importexport.wysiwyg.WysiwygContentType;
 
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
 /**
  * @author tleroi
  */
+@XmlRootElement(name = "publicationContent")
+@XmlAccessorType(XmlAccessType.NONE)
 public class PublicationContentType {
 
+  @XmlElement(name = "wysiwyg")
   private WysiwygContentType wysiwygContentType;
+  @XmlElement(name = "xmlModel")
   private XMLModelContentType xmlModelContentType;
 
+  public PublicationContentType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)